v63003/u_inref/gtm8732nr.csh

```diff
@@ -0,0 +1,61 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+# Test that a line length greater than 8192 bytes produces a LSEXPECTED warning
+#
+
+# Forcing MM, manually setting variables to avoid incompatibilities, rather than letting dbcreate
+# set them with RNG
+setenv gtm_test_db_format "NO_CHANGE"
+setenv acc_meth MM
+setenv test_encryption NON_ENCRYPT
+source $gtm_tst/com/mm_nobefore.csh
+
+echo '# Creating database, setting defer time to 0'
+$gtm_tst/com/dbcreate.csh mumps 1
```

Comment:
Can the dbcreate output be redirected just like dbcheck output. Also please redirect each dbcreate/dbcheck output to a different file _1.txt, _2.txt etc.